client/src/components/SignedInUserImageTile.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react";
+
+const SignedInUserImageTile = (props) => {
+	const [image, setImage] = useState({})
+
+	const getUserImage = async() => {
+		try {
+			const response = await fetch("/api/v1/users/image")
+			if (!response.ok) {
+        throw new Error(`${response.status} (${response.statusText})`)
+			} else {
+				const body = await response.json()
+				setImage(body.image)
+			}
+		} catch(error) {
+			console.error(`Error in fetch: ${error.message}`)
+		}
+	}
+
+	useEffect(() => {
+		getUserImage()
+	}, [])
+
+	let imageDisplay
+	if(image) {
```

Comment:
add space after if

Reply:
ill echo this

Reply:
I thought I changed it and resubmitted! Brain fried.